data/free worlds intro.txt

```diff
@@ -808,6 +808,12 @@ mission "FW Katya 3"
 					goto no
 			
 			label yes
+			`	"Are you being sarcastic?" she asks.
+			choice
+				`	"Yes"
+					goto no
+				`	"No"
```

Comment:
For this and the above, it looks like there are no end ` marks, and they need a period before the end quotation.